Got panic when URL is not accessible

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x78 pc=0x10300f074]

goroutine 1 [running]:
gopkg.in/vansante/go-ffprobe%2ev2.runProbe(0x1400013a000)
	/Users/bastengao/go/pkg/mod/gopkg.in/vansante/go-ffprobe.v2@v2.1.0/ffprobe.go:86 +0x2d4
gopkg.in/vansante/go-ffprobe%2ev2.ProbeURL({0x103060a70?, 0x1400010e0c0}, {0x103019d5f, 0x55}, {0x0?, 0x0, 0x1400006aeb8?})
	/Users/bastengao/go/pkg/mod/gopkg.in/vansante/go-ffprobe.v2@v2.1.0/ffprobe.go:37 +0x254
```

ffprobe -loglevel fatal -print_format json -show_format -show_streams bad_url
```json
{

}
```

Got empty response when got bad URL.

--- a/ffprobe.go
+++ b/ffprobe.go
@@ -82,6 +82,10 @@ func runProbe(cmd *exec.Cmd) (data *ProbeData, err error) {
 		return data, fmt.Errorf("error parsing ffprobe output: %w", err)
 	}
 
+	if data.Format == nil {
+		return data, fmt.Errorf("no format data found in ffprobe output")
+	}
+
 	// Populate the old Tags structs for backwards compatibility purposes:
 	if len(data.Format.TagList) > 0 {
 		data.Format.Tags = &FormatTags{}

